server/controllers/componentsController.test.ts

```diff
@@ -110,224 +44,95 @@ const defaultFooterViewModel = {
   ],
   isPrisonUser: true,
   component: 'footer',
-  services: defaultUserData.services,
 }
 
-const defaultMeta = {
-  activeCaseLoad: {
-    caseLoadId: 'LEI',
-    caseloadFunction: '',
-    currentlyActive: true,
-    description: 'Leeds',
-    type: '',
-  },
-  caseLoads: [
-    {
-      caseLoadId: 'LEI',
-      caseloadFunction: '',
-      currentlyActive: true,
-      description: 'Leeds',
-      type: '',
-    },
-  ],
-  services: [
-    {
-      description: '',
-      heading: 'Service',
-      href: '/href',
-      id: 'service',
-    },
-  ],
+const expectedMeta = {
+  activeCaseLoad: activeCaseLoadMock,
+  caseLoads: [activeCaseLoadMock],
+  services: servicesMock,
 }
 
-describe('getHeaderViewModel', () => {
-  it('should return the HeaderViewModel', async () => {
-    const output = await controller.getHeaderViewModel(defaultUser)
-    expect(output).toEqual(defaultHeaderViewModel)
-    expect(userServiceMock.getUserData).toBeCalledTimes(1)
+describe('componentsController', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
   })
 
-  it('should return empty caseload information if not a nomis user', async () => {
```

Comment:
case load information is now part of `res.locals.user` and therefore provided to the template already